crates/re_viewer_context/src/query_context.rs

```diff
@@ -119,6 +119,32 @@ impl DataResultTree {
             .and_then(|handle| self.data_results.get(handle))
     }
 
+    pub fn first_interesting_root(&self) -> Option<DataResultHandle> {
+        let mut next_handle = self.root_handle();
+
+        while let Some(handle) = next_handle {
+            let node = self.data_results.get(handle).unwrap();
```

Comment:
Please add a note about why this `unwrap` cannot fail (https://github.com/rerun-io/rerun/issues/3408), ideally with a `#![allow(clippy::unwrap_used)]` at the top of the function

Reply:
Better yet, refactored to get rid of the unwrap all together.